site/enroll_service/Dockerfile

```diff
@@ -42,7 +52,11 @@ RUN chmod +x /usr/local/bin/hackedchromedriver
 RUN gem install bundler -v 2.3.24
 
 COPY . $HOME
+
+ARG BUNDLE_GITHUB__COM
```

Comment:
with https://github.com/ideacrew/ea_enterprise/pull/24/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R148, I don't think this and next line are needed (has been working locally without, and test change confirms). I'd assume same for all the following instances of this, if same `environment` item is present in compose file.

Reply:
I tried but it didn't work, the build fails

Reply:
You're totally right. It's been a bit, but I assume I must have been conflating build-time and run-time effects.